safety-architecture/tools/callgraph-tool/scripts/find_coverage_gaps.py

```diff
@@ -18,6 +18,33 @@
 ################################################################################
 
 
+class CallGraphFilter():
```

Comment:
This is currently not used, maybe the class CallGraphFilter could be removed?

Reply:
Removed